fast64_internal/f3d/f3d_gbi.py

```diff
@@ -2107,29 +2034,21 @@ def __init__(self, name, tag, DLFormat):
     def set_addr(self, startAddress, f3d):
         startAddress = get64bitAlignedAddr(startAddress)
         self.startAddress = startAddress
-        print("GfxList " + self.name + ": " + str(startAddress) + ", " + str(self.size(f3d)))
+        print(f"GfxList {self.name}: {str(startAddress)}, {str(self.size(f3d))}")
         return startAddress, startAddress + self.size(f3d)
 
     def save_binary(self, romfile, f3d, segments):
-        print("GfxList " + self.name + ": " + str(self.startAddress) + ", " + str(self.size(f3d)))
+        print(f"GfxList {self.name}: {str(startAddress)}, {str(self.size(f3d))}")
         romfile.seek(self.startAddress)
         romfile.write(self.to_binary(f3d, segments))
 
     def size(self, f3d):
-        size = 0
-        for command in self.commands:
-            size += command.size(f3d)
-        return size
+        return sum([command.size(f3d) for command in self.commands])
 
     # Size, including display lists called with SPDisplayList
     def size_total(self, f3d):
-        size = 0
-        for command in self.commands:
-            if isinstance(command, SPDisplayList) and command.displayList.DLFormat != DLFormat.Static:
-                size += command.displayList.size_total(f3d)
-            else:
-                size += command.size(f3d)
-        return size
+        siz_tot = lambda x: isinstance(command, SPDisplayList) and command.displayList.DLFormat != DLFormat.Static
+        return sum([command.size(f3d) if siz_tot(command) else command.displayList.size_total(f3d) for command in self.commands])
```

Comment:
the ternary is backwards
also would be nice to name the lambda a better name, maybe after "recurse" idk